samples/exceptions/ExceptionSamples.java

```diff
@@ -0,0 +1,35 @@
+package exceptions;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ExceptionSamples {
+    public static void run() {
+
+        int a = 15;
+        int b = 0;
+
+        try {
+            System.out.println(divide(10, 0));
```

Comment:
Variables a and b are not used in this example - shouldn't they be used when calling `divide()` method?